src/NzbDrone.Api/Extensions/RequestExtensions.cs

```diff
@@ -26,5 +26,15 @@ public static bool IsLocalRequest(this Request request)
                     request.UserHostAddress.Equals("127.0.0.1") ||
                     request.UserHostAddress.Equals("::1"));
         }
+
+        public static bool IsLoginRequest(this Request request)
+        {
+            return request.Path.Equals("/login", StringComparison.InvariantCultureIgnoreCase);
+        }
+
+        public static bool IsContentRequest(this Request request)
+        {
+            return request.Path.StartsWith("/Content/", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
Working fine for: IsApiRequest: https://github.com/Sonarr/Sonarr/blob/forms-auth/src/NzbDrone.Api/Extensions/RequestExtensions.cs#L10